modules/statistics/RTPStatsCollector.ts

```diff
@@ -45,24 +93,27 @@ class SsrcStats {
      * Sets the "loss" object.
      * @param loss the value to set.
      */
-    setLoss(loss) {
-        this.loss = loss || {};
+    setLoss(loss: ILoss | undefined): void {
```

Comment:
I don't see `undefined` being passed to this function. You should be able to drop undefined here and also drop default value assignment in the next line.